contracts/ronin/validator/RoninValidatorSet.sol

```diff
@@ -59,7 +59,8 @@ contract RoninValidatorSet is Initializable, CoinbaseExecution, SlashingExecutio
    * deducting amount on slashing).
    */
   function _fallback() internal view {
-    if (msg.sender != stakingVestingContract() && msg.sender != stakingContract()) revert ErrUnauthorizedReceiveRON();
+    if (!(msg.sender == stakingVestingContract() || msg.sender == stakingContract()))
```

Comment:
I think solidity optimizer will handle this part. It's still a good point anyway.

Reply:
Actually, I don't know if short-circuiting is applied in a wrapped `or`: `!(a || b)`? In the current code, `b` will be `a && b` skipped if `a == false`.

Reply:
> Actually, I don't know if short-circuiting is applied in a wrapped `or`: `!(a || b)`? In the current code, `b` in `a && b` will be skipped if `a == false`.

Short-circuiting is still applied. if `a == true`, b will be skipped.

Proposed modification in fact saves 0.004kb since 1 not operation is cut off.